scripts/build/build_musllinux_1_1.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Create musllinux_1_1 wheels for psycopg2
+#
+# Look at the .github/workflows/packages.yml file for hints about how to use it.
+
+set -euo pipefail
+set -x
+
+dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+prjdir="$( cd "${dir}/../.." && pwd )"
+
+# Build all the available versions, or just the ones specified in PYVERS
+if [ ! "${PYVERS:-}" ]; then
+    PYVERS="$(ls /opt/python/)"
+fi
+
+# Don't build musl wheels for Python 3.9 and 3.10 which is the version on
+# Alpine 3.14 and 3.15. The will install but cause 'ImportError' because of
+# conflicting SOAPI values (wanting '-musl.so' but getting '-gnu.so').
+#
+# https://discuss.python.org/t/a-mess-with-soabi-tags-on-musllinux/11688
+PYVERS=${PYVERS//cp39-cp39/}
+PYVERS=${PYVERS//cp310-cp310/}
```

Comment:
This is a bit unfortunate and I suppose something similar should be done for psycopg3. But in my opinion it's better to fail early during installation than at runtime.

Alpine 3.15 seems to actually be using Python 3.9 still, but I think it's probably a good idea to be a bit proactive in case they decide to update to 3.10.

Reply:
Other relevant links:

- https://github.com/nedbat/coveragepy/issues/1268
- https://github.com/python/cpython/pull/24502

Reply:
Seems like this is getting fixed upstream in Alpine and backported to at least 3.15 (hopefully 3.14 as well). How do you want to approach this from a psycopg2 perspective? Just revert this workaround and maybe wait a while before merging to let the Alpine patch to propagate. Or keep the workaround and revert it sometime in the future?

https://gitlab.alpinelinux.org/alpine/aports/-/issues/13227

Reply:
Hmm, seems like it has already been backported to 3.14 and 3.15 🥳

Reply:
> Hmm, seems like it has already been backported to 3.14 and 3.15 :partying_face: 


Yes, tested with `psycopg[binary]` and fresh alpine images and works as expected. I'd say this is no more an issue.